resources/views/components/forum/post.blade.php

```diff
@@ -133,7 +133,7 @@ class="post__author"
             :anon="false"
             :user="$post->user"
         >
-            <x-slot:appended-icons>
+            <x-slot:appendedIcons>
```

Comment:
The documentation states that kebab case should be used in html when referring to component properties: https://laravel.com/docs/10.x/blade#casing